sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/DownloadRetryOptions.java

```diff
@@ -12,10 +12,10 @@
  * This class contains the configuration options used to reliably download from the blob service.
```

Comment:
I don't know the best terminology to use. Is something like this more accurate: "This class contains the configuration options used to download from the blob service using smart retries, starting from a checkpoint, rather than starting over from the beginning."

Reply:
I think that is a good change which help explains the use of the class

Reply:
I've made the change